manifests/rule.pp

```diff
@@ -80,6 +84,22 @@
     fail('Exactly one of "action" or the deprecated "policy" param is required.')
   }
 
+  if $outerface and !($chain in ['FORWARD', 'OUTPUT', 'POSTROUTING']) {
+    fail('Outgoing interface can only be set in the "FORWARD", "OUTPUT" and "POSTROUTING" chains.')
+  } elsif $outerface {
+    $outerface_real = " outerface ${outerface}"
```

Comment:
could you remove the leading whitespace here and instead add it in the `$rule = squeeze()` line?

Reply:
Done. This required some trickery because of the last space before `;` which `squeeze()` doesn't handle.